ansible_bender/builders/buildah_builder.py

```diff
@@ -150,7 +150,10 @@ def configure_buildah_container(container_name, working_dir=None, env_vars=None,
         for k, v in annotations.items():
             config_args += ["--annotation", "%s=%s" % (k, v)]
     if user:
-        config_args += ["--user", user]
+        if os.getuid() != 0:
+            print("Setting user in rootless mode will create issues")
```

Comment:
this will get lost in the output easily, how about:
```python
logger.warning(
  "CHANGING USER IN ROOTLESS MODE IS DISCOURAGED, "
  "USE IT ONLY WHEN YOU KNOW WHAT YOU'RE DOING, "
  "YOU'VE BEEN WARNED"
)
```
and now that I'm looking at it, I'd probably drop the `else:` branch - I'd say that we can let users shoot themselves in the foot if they want - we've just warned them it will hurt